activerecord/test/activejob/destroy_association_async_test.rb

```diff
@@ -209,6 +210,20 @@ class DestroyAssociationAsyncTest < ActiveRecord::TestCase
     DestroyAsyncParent.delete_all
   end
 
+  test "polymorphic belongs_to" do
```

Comment:
@fatkodima FYI, this test is failing pretty consistently for me locally:

```
$ bin/test --seed 16437

Failure:
DestroyAssociationAsyncTest#test_polymorphic_belongs_to [test/activejob/destroy_association_async_test.rb:219]:
`Author.count` didn't change by -1, but by -2.
Expected: 3
  Actual: 2


bin/test test/activejob/destroy_association_async_test.rb:213
```

Have tried with mysql2, postgres, and sqlite3_mem.

Reply:
Thanks for reporting, I will take a look.